### PR TITLE
Add bitswap commands

### PR DESCRIPTION
**Pull request by fredthomsen**
Adding bitswap command as suggested by #42.

Closing #47.

### CODE REVIEW

**Comment by whereswaldon** [comment]
The code looks good, thanks. Could you write a quick test for each of the bitswap functions? If you follow the general model of the tests in 'test/functional/tests.py', it should be pretty easy.

**Comment by fredthomsen** [comment]
In-progress.  Should have mentioned that above.

**Comment by fredthomsen** [comment]
Added the tests. Passing on my box but Travis seems to be taking a bit.

**Comment by whereswaldon** [comment]
The entire IPFS organization on Github shares one Travis instance. We
queue up behind whatever builds are going on for each of the other
projects that IPFS hosts. It's unfortunate, and they're working to fix
it, but for now we have to wait... sigh.

**Comment by fredthomsen** [comment]
Guess waiting until morning is key.  Life is good.

File: test/functional/tests.py [hunk 1]
```diff
@@ -492,5 +492,28 @@ def test_object_patch_set_data(self):
         self.assertEqual(result,
                 {'Hash': 'QmV4QR7MCBj5VTi6ddHmXPyjWGzbaKEtX2mx7axA5PA13G'})
 
+
```
Comment (whereswaldon) [inline test/functional/tests.py]: @alexander255 just updated our testing framework to skip the functional tests if an IPFS daemon isn't running. It's a one-line change to make your tests have this feature. Take a look at [this class decorator](https://github.com/ipfs/py-ipfs-api/pull/57/files#diff-322d04fe8e037003724bccda9930e89eR49). If you could just throw that on this test class, I'm good to merge this.

**Comment by alexander255** [comment]
:+1: on this change once that decorator has been added.

**Comment by fredthomsen** [comment]
Done.

**Comment by whereswaldon** [comment]
@fredthomsen Thanks a ton for putting this together! Sorry that we kept adding todos as you went.

**Pull request merged**